packages/gestalt/src/MenuOption.js

```diff
@@ -132,12 +124,14 @@ export default function MenuOption({
       </Box>
       {isExternal && (
         <Box
-          display="flex"
+          alignItems="center"
           color="transparent"
+          display="flex"
           justifyContent="center"
-          alignItems="center"
+          // marginStart is for spacing relative to Badge, should not be moved to parent Flex's gap
           marginStart={2}
         >
+          {/* TODO: this label needs to be translated */}
```

Comment:
This probably works until we figure out a story for translation in Gestalt, but it's something we should address then